### PR TITLE
set ActiveProcessorCount based on ...requests.cpu

### PR DESCRIPTION
**Pull request by domruf**
Instead of telling the user to manually update ActiveProcessorCount,
simply add the setting automatically based on
.Values.jira.resources.container.requests.cpu.

### CODE REVIEW

File: src/main/charts/jira/templates/config-jvm.yaml [hunk 1]
```diff
@@ -10,6 +10,11 @@ data:
     {{- range .Values.jira.additionalJvmArgs }}
     {{ . }}
     {{- end }}
+    {{- if hasSuffix "m" .Values.jira.resources.container.requests.cpu }}
+    -XX:ActiveProcessorCount={{ div (trimSuffix "m" .Values.jira.resources.container.requests.cpu) 1000 | default 1 }}
+    {{ else }}
+    -XX:ActiveProcessorCount={{ .Values.jira.resources.container.requests.cpu }}
+    {{ end }}
```
Comment (nanux) [inline src/main/charts/jira/templates/config-jvm.yaml:17]:
```suggestion
    {{- end }}
```
to remove the empty line

**Comment by nanux** [comment]
@domruf, thank you for the improvements. This will make the configuration less error-prone. Would you mind adding the same implementation to the other products as well?

Can you also fix the `test.RequestsTest.sts_resource_requests_and_limits` test please? It would be great to add some additional unit tests to cover this logic.

**Comment by domruf** [comment]
@nanux okay I'll look into it.

**Review by nanux — APPROVED** [review]

**Review by nghazali — APPROVED** [review]

**Comment by nanux** [comment]
Thanks @domruf !

**Pull request merged**